docs/body.md

```diff
@@ -748,39 +615,336 @@ when this happens.
+
+            - disk.read.bytes
+            - disk.read.requests
+            - disk.write.bytes
+            - disk.write.requests
+
+    - Ensure the resulting YAML syntax is valid restart the ceilometer-polling service
+
+    Finally note that these metrics ARE deprecated and will be removed in
+    future releases of OpenStack itself.
+
+    Note: **There is no known template configuration for this for RHOSP
+          templates at this time and are unsupported.**
+
+*   Make sure you restart ceilometer-notification service whenever you change
+    `pipeline.yaml` or `event_pipeline.yaml`.
+
+*   For Queens and Rocky devices, make sure to restart ceilometer-polling service
+    if you have updated `/etc/ceilometer/polling.yaml`
+
+*   The RabbitMQ-Ceilometer service
+
+    `RabbitMQ-Ceilometer` is typically only needed for legacy (pre 3.0,0 ZenPack) systems.
+
+    Version 2.4.0 of this zenpack introduces a new service, `RabbitMQ-Ceilometer`.
+    This is a dedicated instance of RabbitMQ on each collector which is used solely
+    for integration with Ceilometer, rather than using the standard `RabbitMQ`
+    service that is used by Zenoss itself. This better distributes any load as
+    well as providing better support for distributed collector scenarios where the
+    target OpenStack environment might not have network access to the central
+    Zenoss servers.
+
+*   Zenoss collectors need direct access to the NovaHost devices in order to
+    model them. In some setups like RHOSP's overcloud, where direct
+    access to the overcloud network is not accessible by default, this can be a
+    challenge. Discussion of how to achieve connectivity to overcloud is
+    out of scope.
+
+### Optional Steps
+
+#### VM State Changes
+
+By default, instance state changes will be captured by Zenoss when
+certain events occur, for example, when an instance is shut down, the
+state change to SHUTDOWN will be reflected in Zenoss.
+
+However, certain state changes that don't correspond to another defined
+event may not be picked up until the next time Zenoss models the
+environment.
+
+If you would like to reduce the likelihood of this occurring, you can
+configure OpenStack Nova to send an event (through Ceilometer) to Zenoss
+whenever any VM state change occurs by adding the following to
+/etc/nova/nova.conf on all Nova hosts:
+
+      [notifications]
+      notify_on_state_change=vm_and_task_state
+
+      ...
+
+      [oslo_messaging_notifications]
+      driver = messagingv2
+
+Save /etc/nova/nova.conf and restart nova services.
+
+Note that notify_on_state_change will cause increased event load,
+both on OpenStack and Zenoss, and additional processing within the event
+transforms in Zenoss to keep the model consistent. Since most instance
+changes will still be caught without this option enabled, it is
+recommended to leave notify_on_state_change disabled if your
+OpenStack environment is very large.
+
+#### VM State Changes RHOSP
+
```

Comment:
i'd just omit this.  If we move the other part into the manual config section, it not being mentioned in the RHOSP instructions is sufficient